modules/types_proto/proto/hephaestus/types/proto/dummy_type.proto

```diff
@@ -6,7 +6,7 @@ syntax = "proto3";
 
 package heph.types.proto;
 
-enum DummyTypeDummyEnum {
+enum DummyTypeExternalDummyEnum {
   A = 0;
   B = 1;
   C = 2;
```

Comment:
To follow `buf` requirement, as proto enum are not class, it forces you to append the class name as prefix:
```suggestion
enum DummyTypeExternalDummyEnum {
  DUMMY_TYPE_EXTERNAL_DUMMY_ENUM_A = 0;
  B = 1;
  C = 2;
```
see https://github.com/olympus-robotics/hephaestus/pull/162/files#diff-e47d59905961e3347b18223422fce7ea857673877a9e75a4ffa52d0da6c2bee7R8

Reply:
We will enforce this once we introduce buff, hopefully soon. So I think we can start by now updating the enum names.
You have that nice camel case to scream case function so it should be easy to add the prefix.

Reply:
Ok. But are you sure about the case? If it is a nested enum, proto will create DummyType_InternalDummyEnum_A automatically.

Reply:
Yes, I didn't try for nested enums but for external one yes. I mean we can leave it as it is now and then when I finish the buf PR I will update your prefix function

Reply:
Let's do that change at once. We will also have to update forkify, so let's do that in one push and leave the current implementation.

Reply:
I just realized that this is only a change in the proto files, not in the cpp enums. I implemented your suggestion in this MR.